cpp/oneapi/dal/backend/primitives/reduction/BUILD

```diff
@@ -0,0 +1,12 @@
+package(default_visibility = ["//visibility:public"])
+load("@onedal//dev/bazel:dal.bzl", "dal_module")
+
+dal_module(
+    name = "reducer",
```

Comment:
Rename to match directory name -> reduction

Reply:
Fixed